guides/common/modules/proc_exporting-a-content-view-version.adoc

```diff
@@ -14,7 +14,7 @@ ifdef::client-content-dnf[]
 {ProjectServer} exports only RPM, kickstart files, and Docker content added to a version of a content view.
 endif::[]
 ifndef::client-content-dnf[]
-{ProjectServer} exports only RPM, kickstart files, DEB content, and Docker content added to a version of a content view.
+{ProjectServer} exports only RPM, kickstart files, Deb content, and Docker content added to a version of a content view.
```

Comment:
I recommend rewriting this to avoid anthropomorphism. Example: "You can only export ..."

Reply:
Reworded.